src/js/actions/policy.js

```diff
@@ -189,6 +213,61 @@ define(function (require, exports) {
     removePointerPolicies.writes = [locks.PS_APP, locks.JS_POLICY];
 
     /**
+     * Temporarily disable keyboard policies, caching the previous state
+     *
+     * @return {Promise}
+     */
+    var disableKeyboardPolicies = function () {
+        var policyStore = this.flux.store("policy"),
+            kind = PolicyStore.eventKind.KEYBOARD;
+
+        _cachedKeyboardPolicies = policyStore.getPolicies(kind);
```

Comment:
Maybe in the spirit of making this hard to screw up, check here to make sure there aren't already cached keyboard policies? I.e., no double disabling.